src/components/button/Button.tsx

```diff
@@ -89,6 +89,7 @@ const AminoButton = styled.button<ButtonProps<GroupTag>>`
   cursor: pointer;
   white-space: nowrap;
   padding: ${p => getPadding(p.size)};
+  width: fit-content;
```

Comment:
The button currently is in "display:flex", so obviously it will need to have a wrapper or a flex parent to handle the width. If you want it to fit the button content. What do you think about use `inline-flex` instead 🤔 ?

Reply:
It has nothing to do with flex from what I can see. I changed it to `block` and it didn't change the width.